### PR TITLE
fix WebComponents mangling bug

### PR DESCRIPTION
**Pull request by arilotter**
closes https://github.com/terser/terser/issues/1170

### CODE REVIEW

**Comment by Brotenko** [comment]
This should probably also include `disconnectedCallback` and `adoptedCallback` - See documentation Web Component documentation [here](https://developer.mozilla.org/en-US/docs/Web/API/Web_components#custom_elements_2)

File: tools/domprops.js [hunk 1]
```diff
@@ -3638,6 +3639,7 @@ export var domprops = [
     "connectShark",
     "connectStart",
     "connected",
+    "connectedCallback',
```
Comment (Brotenko) [inline tools/domprops.js:3642]:
```suggestion
    "connectedCallback",
```

**Review by Brotenko — CHANGES_REQUESTED** [review]

**Comment by fabiosantoscode** [comment]
Cool, thanks! I'll have a look around this doc page and look for more properties there.

**Comment by fabiosantoscode** [comment]
I merged this now but I only release during the week, in order to avoid weekend breakage.

**Comment by arilotter** [comment]
yo  @fabiosantoscode the PR had a bug!! typo with wrong quotes 😭 
PR to fix that + two more webcomponent issues https://github.com/terser/terser/pull/1525

**Pull request merged**